src/index.html

```diff
@@ -1,17 +1,151 @@
-<!doctype html>
+<!doctype  html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+  action="http://localhost:3000/api"
+  method="post"
+  onsubmit="onSubmit()"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label for="surname" class="form-field">
+        Surname:
```

Comment:
Зроби відступ для кращої читабельності
```suggestion
        <label for="surname" class="form-field">
          Surname:
```